Result/Result.swift

```diff
@@ -140,7 +173,7 @@ public func materialize<T>(_ f: () throws -> T) -> Result<T, AnyError> {
 
 @available(*, deprecated, renamed: "Result.init(_:)")
 public func materialize<T>(_ f: @autoclosure () throws -> T) -> Result<T, AnyError> {
-	return Result(f)
+	return Result(try f())
```

Comment:
This is needed (for Swift 5) because of the error: `error: add () to forward @autoclosure parameter`.